packages/strapi-plugin-upload/services/image-manipulation.js

```diff
@@ -42,6 +42,7 @@ const generateThumbnail = async file => {
       const { width, height, size } = await getMetadatas(newBuff);
 
       return {
+        name: `thumbnail_${file.name}`,
```

Comment:
the filename might be really strange here has it can be set by the user. What was the need for it here ?

Reply:
The name attribute should be set for all media files because it goes to the same handler for upload. A user can use file name as a property for a media library in custom plugin.
For example, to provide uniqueness of file names in a library. Only hash for this is not enough because it is random generated every time.